### PR TITLE
Cange skills to pills

### PR DESCRIPTION
**Pull request by LiamNotorp1985**

### CODE REVIEW

File: How to learn new language.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ develop they articulation.
  trying to  decrease amount of mistakes as much as
  possible and keep practicing using the most powerful instrument 
  we ever could get will definitely lead as to systematic accumulation 
- and reinforcement of our language skill.
```
Comment (LiamNotorp1985) [inline How to learn new language.md]: Thats obligatory

Comment (dupliaka) [reply How to learn new language.md]: Congrats with your first commit!

Here some points to remember:

1.Before steps

- Check that you in local master (bottom right corner->Local branches/master->checkout)
*You can compare local master and upstream  (bottom right corner->upstream branches/master->compare with current)
- Merge upstream with master VCS->pull-> upstream/master


2.Push to upstream

Update your state with upstream and add your changes to your origin.

It is hardly recommended to have small changes for every commit(without push)
So, do your job, commit, change something - commit...

When you finish work on your task: 

- Go to master  (bottom right corner->Local branches/master->checkout)
- Get upstream changes VCS->pull-> upstream/master
- Go to feature branch (bottom right corner->Local branches/master->checkout)
- Merge it with local updated master (vcs->merge master local)
- Do push (vcs ->push) .

3. Update origin
 
When you make a push you did it on your feature branch, so you need to merge it with master origin in the end: 

- checkout master  (bottom right corner->Local branches/master->checkout)
- merge feature branch to master (bottom right corner -> feature branch -> merge into current)
- push changes vcs->push origin

**Comment by dupliaka** [comment]
Please, take a look at the first article if that contains any problems.

File: How to learn new language.md (hunk 1 above)
Comment (LiamNotorp1985) [reply How to learn new language.md]: Thanks for reminder

**Pull request merged**